src/core/serpents.erl

```diff
@@ -51,13 +54,14 @@ start_phase(start_cowboy_listeners, _StartType, []) ->
     , cowboy_swagger_handler
     ],
   Trails =
-    [ {"/", cowboy_static, {file, "www/index.html"}}
-    , {"/favicon.ico", cowboy_static, {file, "www/assets/favicon.ico"}}
-    , {"/assets/[...]", cowboy_static, {dir, "www/assets"}}
-    , {"/game/:game_id", cowboy_static, {file, "www/game.html"}}
+    [ trails:trail("/", cowboy_static, {file, "www/index.html"})
+    , trails:trail(
+        "/favicon.ico", cowboy_static, {file, "www/assets/favicon.ico"})
+    , trails:trail("/assets/[...]", cowboy_static, {dir, "www/assets"})
+    , trails:trail("/game/:game_id", cowboy_static, {file, "www/game.html"})
     | trails:trails(Handlers)
     ],
-  trails:store(Trails),
+  ok = trails:store(Trails),
```

Comment:
According to **Dialyzer**:
>  The call trails:store(Trails:[{'_' | binary() | maybe_improper_list(binary() | maybe_improper_list(any(),binary() | []) | byte(),binary() | []),atom() | tuple(),_} | {'_' | binary() | maybe_improper_list(binary() | maybe_improper_list(any(),binary() | []) | byte(),binary() | []),[{atom(),'int'} | {atom(),'function',fun((binary()) -> boolean() | {'true',_})}],atom() | tuple(),_} | trails:trail(),...]) will never return since the success typing is ([{'_' | binary() | maybe_improper_list(binary() | maybe_improper_list(any(),binary() | []) | byte(),binary() | []),atom() | tuple(),_} | {'_' | binary() | maybe_improper_list(binary() | maybe_improper_list(any(),binary() | []) | byte(),binary() | []),[{_,_} | {_,_,_}],atom() | tuple(),_}]) -> 'ok' and the contract is ([trail()]) -> 'ok'